addons/kotsadm/alpha/tmpl-kotsadm-cacerts.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kotsadm
+  labels:
```

Comment:
Could the labels be left off?

Reply:
the other kotsadm patch has labels there, and I felt that matching what it did was appropriate